packages/cactus-test-geth-ledger/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@hyperledger/cactus-test-geth-ledger",
+  "version": "2.0.0-alpha.1",
+  "description": "Helpers for running test go-ethereum ledger in test scripts.",
+  "keywords": [
+    "Hyperledger",
+    "Cactus",
+    "Integration",
+    "Blockchain",
+    "Distributed Ledger Technology"
+  ],
+  "homepage": "https://github.com/hyperledger/cactus#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger/cactus/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/cactus.git"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Hyperledger Cactus Contributors",
+    "email": "cactus@lists.hyperledger.org",
+    "url": "https://www.hyperledger.org/use/cactus"
+  },
```

Comment:
@rwat17 Could you please update these to refer to the project as cacti not cactus. I also just noticed this, sorry for missing it in the previous review.

Reply:
changed